### PR TITLE
Update endpoint-dlp-learn-about.md

### PR DESCRIPTION
**Pull request by msbemba**
Copying sensitive content to the clipboard in public preview -https://techcommunity.microsoft.com/t5/microsoft-security-and/announcing-public-preview-of-microsoft-endpoint-data-loss/ba-p/1534085 
"Modifying a file" activity is also available in Endpoint DLP

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3399

File: microsoft-365/compliance/endpoint-dlp-learn-about.md [hunk 1]
```diff
@@ -40,7 +40,9 @@ Microsoft Endpoint DLP enables you to audit and manage the following types of ac
 |copied to network share, e.g. \\my-server\fileshare   |     auditable and restrictable    |
 |printed |    auditable and restrictable       |
 |copied to cloud via Chromium Edge    |   auditable and restrictable        |
+|copied to clipboard   |   auditable and restrictable      |
 |accessed by unallowed apps and browsers    |  auditable and restrictable       |
+|Modify a file    |  auditable and restrictable       |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-learn-about.md]:
```suggestion
|modify a file    |  auditable and restrictable       |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
separate update made, this PR is outdated

**Pull request closed without merging**